bin/kintsugi

```diff
@@ -26,14 +28,20 @@ end
 
 def create_options_parser
   OptionParser.new do |opts|
-    opts.banner = "Usage: kintsugi [pbxproj_filepath] [options]"
+    opts.banner = "Kintsugi, version #{Kintsugi::Version::STRING}\nCopyright (c) 2021 " \
+                  "Lightricks\n\nUsage: kintsugi [pbxproj_filepath] [options]"
     opts.on("--changes-output-path=PATH", "Path to which changes applied to the project are " \
             "written in JSON format. Used for debug purposes.")
 
     opts.on("-h", "--help", "Prints this help") do
       puts opts
       exit
     end
+
+    opts.on_tail("-v", "--version", "Show version") do
```

Comment:
to have consistent verb conjugation, either change to `Shows`, or change `Prints` above to `Print`.